code/datums/trading/traders/goods.dm

```diff
@@ -207,7 +207,7 @@ Sells devices, odds and ends, and medical stuff
 								/obj/item/uv_light                  = TRADER_THIS_TYPE,
 								/obj/item/mmi                       = TRADER_ALL,
 								/obj/item/robotanalyzer             = TRADER_THIS_TYPE,
-								/obj/item/toner                     = TRADER_THIS_TYPE,
+								/obj/item/toner_cartridge                     = TRADER_THIS_TYPE,
```

Comment:
Spacing oh no

Reply:
gawd, that's definitely something using tabs for spacing. -_-